Makefile

```diff
@@ -283,14 +285,18 @@ web: web-pre-process $(HUGO)
 	# TODO(bwplotka): Make it --gc
 	@cd $(WEB_DIR) && HUGO_ENV=production $(HUGO) --config hugo.yaml --minify -v -b $(WEBSITE_BASE_URL)
 
+.PHONY: failint
+failint:
```

Comment:
Please let's add this to `lint` command. We already have too many commands ;p

Reply:
I I know you added this as lint dep, but I  think it might be more explicit when inlined in `lint` command.

Reply:
It makes moving it inside of the `lint` command although then we would have to wait for the whole linting to complete whenever we want to test (not a huge deal).

Reply:
I think that's ok (:

Reply:
You can put it in front of it if you think it's fast.

Reply:
I think I've fixed this, please let me know if you were thinking something different when you said to put it in front.